backend/src/main/java/no/ntnu/database/controller/CategoryController.java

```diff
@@ -56,21 +56,20 @@ public Iterable<Category> getAllCategories() {
 	}
 
 	/**
-	 * Endpoint to search for categories.
+	 * Endpoint to find a category.
 	 *
-	 * @param query The query to use when searching for categories.
-	 *
-	 * @return Categories that match the search queries
+	 * @param id The ID of the category
+	 * @return 200 with a category if found, 404 otherwise
 	 */
-	@Operation(summary = "Search for categories",
-			description = "Search for categories based on a database query string")
+	@Operation(summary = "Get a category",
+			description = "Get a category from the database")
 	//TODO: Improve swagger here with api responses.
-	@ApiResponse(responseCode = "200", description = "Successfully searched")
-	@GetMapping(value = "{query}", produces = {"application/json"})
-	public Iterable<Category> searchCategory(@PathVariable String query) {
-		return query == null || query.isBlank()
-				? service.getAllCategories()
-				: service.searchCategory(query);
+	@ApiResponse(responseCode = "200", description = "Found category")
+	@GetMapping(value = "{id}", produces = {"application/json"})
+	public ResponseEntity<Category> searchCategory(@PathVariable int id) {
```

Comment:
If you're no longer searching, should this perhaps be renamed to getCategory instead?

Reply:
Yes, I fixed it :)